CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ cmake_minimum_required(VERSION 3.25)
 project(TheoIDE VERSION 0.1.0 LANGUAGES CXX)
 
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CPACK_PACKAGE_VERSION_MAJOR "0")
```

Comment:
I think it would be better to not repeat the version parts, but using the variables already set by CMake like `CMAKE_PROJECT_VERSION_MAJOR`